vrecord

```diff
@@ -774,8 +774,8 @@ GTKFORM
 export MAIN_DIALOG
 }
 
-  HOME_DIALOG=$(cat << GTKFORM
-  <window title="Welcome to Vrecord!">
+HOME_DIALOG=$(cat << GTKFORM
+<window title="Welcome to Vrecord!">
```

Comment:
Do we write `Vrecord` with a capital initial or `vrecord` all in lowercase letters?